frontend/pages/doors/edit.tsx

```diff
@@ -18,16 +18,27 @@ import Link from '~/components/Link';
 import StrongConfirmDialog from '~/components/StrongConfirmDialog';
 import genGetProps from '~/functions/genGetServerSideProps';
 import { useGetDoorsQuery, useRemoveDoorMutation } from '~/generated/graphql';
+import { hasAccess, useApiAccess } from '~/providers/ApiAccessProvider';
 import { useSetPageName } from '~/providers/PageNameProvider';
 
 export default function EditDoorsPage() {
   const { t } = useTranslation();
+
   useSetPageName(t('doors:editDoorAccess'));
+  const apiContext = useApiAccess();
 
   const [openDialog, setOpenDialog] = useState<string>('');
 
   const { data, refetch: refetchDoors } = useGetDoorsQuery();
   const [removeDoor] = useRemoveDoorMutation();
+
+  if (!hasAccess(apiContext, 'core:access:door:read')
+   && !hasAccess(apiContext, 'core:access:door:create')
+   && !hasAccess(apiContext, 'core:access:door:update')
+   && !hasAccess(apiContext, 'core:access:door:delete')) {
+    return <h2>You do not have access to this page.</h2>;
```

Comment:
I know it's unnecessary, but using the translation strings we already have is always nice.